app/src/play/res/layout/media_router_controller.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <!--<ImageView android:id="@+id/mrc_art"-->
+        <!--android:layout_width="match_parent"-->
+        <!--android:layout_height="wrap_content"-->
+        <!--android:adjustViewBounds="true"-->
+        <!--android:scaleType="fitCenter"-->
+        <!--android:background="?attr/colorPrimary"-->
+        <!--android:layout_gravity="top"-->
+        <!--android:visibility="gone"/>-->
```

Comment:
Feel free to delete things you don't need anymore. We can always get it back from the repo history later.